account_invoice_triple_discount/models/account_move.py

```diff
@@ -39,6 +37,6 @@ def _has_discount(self):
         return any(
             [
                 line._compute_aggregated_discount(line.discount) > 0
-                for line in self.invoice_line_ids
+                for line in self.line_ids
```

Comment:
This check was on `line_ids`, why is it changed? Does not seem to affect the added test result